NKalkan/KalkanApi.cs

```diff
@@ -181,6 +181,10 @@ public void ValidateCertificate(string certificate, KalkanValidationType validat
             ThrowIfError(errorCode);
         }
 
+        // Fix: BUFFER_TOO_SMALL
```

Comment:
So you are saying that Kaplan report incorrect value? Did you have Latin text there or is there some Cyrillic characters?

Reply:
If you do not increase the value, the message BUFFER_TOO_SMALL is constantly displayed. I looked at your changes (+20) - but it doesn't work.

Reply:
`outputInformation:` OCSP: check certificate status: good
	certificateSerialNumber=50B4B1547BEB730C70CEBB817F3FE802E384704B
	This Update: 18.01.2024 14:57:09 ALMT
Verify chain and certificates: - OK.

`ospResponce`: OCSP Response Data:
    OCSP Response Status: successful (0x0)
    Response Type: Basic OCSP Response
    Version: 1 (0x0)
    Responder Id: CN = OCSP RESPONDER, C = KZ, O = \D2\B0\D0\9B\D0\A2\D0\A2\D�g���

Reply:
Let's duplicate on 2. I think that would be slightly more safe in case they count number of Cyrillic characters, but forgte take into account encoding.

Reply:
I checked the full length of `ospResponseBuilder` - its value is 8006. Looked at the test cases in SDK (Test_const.cpp) and they have a length value of 8192 (LENGHT_STAND global variable). Don't know how to calculate correctly. With this length, `ospResponseBuilder` does not cause errors, it simply cuts out, but `outputInformationBuilder` is critical, if it is not enough, then the `BUFFER_TOO_SMALL` error occurs.

![image](https://github.com/kant2002/NKalkan/assets/53144972/57934326-74ff-4544-a652-b870461e2892)